app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -34,13 +34,12 @@ def update
       if result.success?
         if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
           idv_session.opted_in_to_in_person_proofing = false
-          idv_session.skip_doc_auth = false
+          idv_session.skip_doc_auth_from_handoff = false
```

Comment:
I understand removing `idv_session.skip_doc_auth = false`.  I'm not sure why `idv_session.skip_doc_auth_from_handoff = false` was added here.